pytype/pytd/typeshed.py

```diff
@@ -104,20 +104,26 @@ def _load_stdlib_versions(self):
     """Loads the contents of typeshed/stdlib/VERSIONS.
 
     VERSIONS lists the stdlib modules with the Python version in which they were
-    first added, in the format `{module}: {major}.{minor}`. Note that this file
-    ignores the stdlib/@python2 subdirectory! If stdlib/foo.pyi targets Python
-    3.6+ and stdlib/@python2/foo.pyi, 2.7, VERSIONS will contain `foo: 3.6`.
+    first added, in the format `{module}: {min_major}.{min_minor}-` or
+    `{module}: {min_major}.{min_minor}-{max_major}.{max_minor}`. Note that this
+    file ignores the stdlib/@python2 subdirectory! If stdlib/foo.pyi targets
+    Python 3.6+ and stdlib/@python2/foo.pyi, 2.7, VERSIONS will contain
+    `foo: 3.6-`.
 
     Returns:
-      A mapping from module name to (major, minor) Python version.
+      A mapping from module name to ((min_major, min_minor),
+      (max_major, max_minor)) Python version. The max tuple can be `None`.
     """
     _, text = self._load_file(os.path.join("stdlib", "VERSIONS"))
     versions = {}
     for line in text.splitlines():
-      match = re.fullmatch(r"(.+): (\d)\.(\d+)", line)
```

Comment:
While you're at it could you add support for skipping blank and comment lines here, like mypy does?